pom.xml

```diff
@@ -10,7 +10,24 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.baloise.confluence</groupId>
     <artifactId>digital-signature</artifactId>
-    <version>7.0.5</version>
+    <version>8.0.0</version>
```

Comment:
Shouldn't we use 8.0.0-SNAPSHOT here until we build a release? Don't know how Confuence will behave when we try to override version 8.0.0 with a "new" build of version 8.0.0 ...

Reply:
Confluence doesn't care about overriding v 8.0.0 with 8.0.0 as long as the new version number is >= the present one, it works